scripts/build.js

```diff
@@ -0,0 +1,110 @@
+                  + '<metadata>CC0 1.0 Universal | simple-icons contributors</metadata>\n'
+                  + '<defs>\n'
+                  + '<font id="Simple Icons" horiz-adv-x="1200">\n'
+                  + '<font-face font-family="Simple Icons" units-per-em="1200" ascent="-1" descent="1200" />\n'
+                  + '<missing-glyph horiz-adv-x="0" />\n';
+  
+  let startUnicode = 0xea01;
+  let usedUnicodes = [];
+  
+  let unicodeHexBySlug = {}
+  
+  for (let iconTitle in SimpleIcons) {
+    let unicode = ucs2.decode(
+      String.fromCodePoint(startUnicode++)).map(
+        point => '&#x' + point.toString(16).toUpperCase() + ';')[0];
+    if (usedUnicodes.includes(unicode)) {
+      throw Error(`Unicodes must be unique. Found '${unicode}' repeated`)
+    }
+    
+    let icon = SimpleIcons[iconTitle];
+    const verticalTransformedPath = SVGPath(icon.path).scale(50, -50).toString();
+    
+    fileContent += `<glyph glyph-name="${icon.slug}" unicode="${unicode}" d="${verticalTransformedPath}" horiz-adv-x="1200" />\n`;
+    usedUnicodes.push(unicode);
+
+    unicodeHexBySlug[icon.slug] = {
+      unicode: unicode,
+      hex: icon.hex
+    };
+  }
+  fileContent += '</font>\n</defs>\n</svg>\n';
+  
+  const outputFilePath = path.join(DISTDIR, 'SimpleIcons.svg');
+  fs.writeFileSync(outputFilePath, fileContent);
+  console.log(`'${outputFilePath}' file built`);
+  
+  return unicodeHexBySlug;
+}
+
+const buildSimpleIconsCssFile = (unicodeHexBySlug) => {
+  let simpleIconsCss = fs.readFileSync(
+      path.resolve(__dirname, '..', 'preview', 'css', 'base.css'));
+  
+  for (let slug in unicodeHexBySlug) {
+    let icon = unicodeHexBySlug[slug];
+    
+    // &#xF26E; -> \f26e
+    const cssDecodedUnicode = icon.unicode.replace('&#x', '\\').replace(';', '').toLowerCase();
+    simpleIconsCss += '\n'
+                    +   `.simpleicons-${slug}::before { content: "${cssDecodedUnicode}"; }\n`
+                    +   `.simpleicons-${slug}.simpleicons--color::before { color: #${icon.hex}; }\n`;
+  }
+  
+  const outputFilePath = path.join(DISTDIR, 'SimpleIcons.css');
+  fs.writeFileSync(outputFilePath, simpleIconsCss);
+  console.log(`'${outputFilePath}' file built`);
+}
+
+const buildSimpleIconsTtfFontFile = () => {
+  const ttf = svg2ttf(fs.readFileSync(path.join(DISTDIR, 'SimpleIcons.svg'), 'utf8'), {});
+  const outputFilePath = path.join(DISTDIR, 'SimpleIcons.ttf');
+  fs.writeFileSync(outputFilePath, new Buffer(ttf.buffer));
+  console.log(`'${outputFilePath}' file built`);
+}
+
+const buildSimpleIconsWoffFontFile = () => {
+  var ttf = new Uint8Array(fs.readFileSync(path.join(DISTDIR, 'SimpleIcons.ttf')));
+  var woff = new Buffer(ttf2woff(ttf).buffer);
+  const outputFilePath = path.join(DISTDIR, 'SimpleIcons.woff');
+  fs.writeFileSync(outputFilePath, woff);
+  console.log(`'${outputFilePath}' file built`);
+}
+
+const buildSimpleIconsWoff2FontFile = () => {
+  const input = fs.readFileSync(path.join(DISTDIR, 'SimpleIcons.ttf'));
```

Comment:
Would prefer not having to read this after it has been created and instead just pass the `SimpleIcons.ttf` content as an argument.